docs/migrate-to-antrea.md

```diff
@@ -0,0 +1,75 @@
+# Migrate from other CNI to Antrea
+
+This document describes how to migrate from other CNIs to Antrea.
+
+NOTE: The following is a reference list of CNIs and versions we have already
+verified the migration process. CNIs and versions that are not listed here
+might also work, please create an issue if you run into problems during the
+migration to Antrea. In addition, during the migration process, no Kubernetes
+resources should be created or deleted, otherwise the migration process might
+fail or some unexpected problems might occur.
+
+| CNI     | Version |
+|---------|---------|
+| Calico  | v3.26   |
+| Flannel | v0.22.0 |
+
+The migration process is divided into three steps:
+
+1. Clean up old CNI.
```

Comment:
the old CNI

Reply:
"The" Added.

Reply:
You probably missed this one.

Reply:
Sorry, thanks for the reminder. Addressed.